packages/auth/src/access/types.ts

```diff
@@ -125,20 +125,12 @@ export interface IncomingPaymentLimit {
 }
 
 export type OutgoingPaymentLimit = {
+  receiver: string
   sendAmount?: PaymentAmount
   receiveAmount?: PaymentAmount
   description?: string
   externalRef?: string
```

Comment:
These were removed from the outgoing payment limit
https://github.com/interledger/open-payments/pull/144/files#diff-aaff56ee0c5bb1e404fc4e010a7e38034fbc755542d28199e4cf0acb5ef580c9L627-L630
```suggestion
```